src/doc/unstable-book/src/compiler-flags/default-visibility.md

```diff
@@ -0,0 +1,44 @@
+# `default-visibility`
+
+The tracking issue for this feature is: https://github.com/rust-lang/compiler-team/issues/782
```

Comment:
I know it's pre-existing but this needs a real tracking issue, the compiler MCP issue is not a tracking issue.

Could you create one and link it here ;-)